dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/DataSourceServiceImpl.java

```diff
@@ -139,6 +140,8 @@ public Result<Object> createDataSource(User loginUser, BaseDataSourceParamDTO da
         } catch (DuplicateKeyException ex) {
             logger.error("Create datasource error.", ex);
             putMsg(result, Status.DATASOURCE_EXIST);
+        } catch (RuntimeException e) {
+            throw new RuntimeException(e.getMessage());
         }
```

Comment:
Please don't hide the exception stack.
```suggestion
        }
```